### PR TITLE
Add background view

### PR DESCRIPTION
**Pull request by hmeriann**
a backgroundView added to the OverviewContactViewController and set its color to tertiarySystemBackground color after check with the Dark appearance
Screenshot 2024-04-15 at 13 16 05

### CODE REVIEW

**Comment by pavoleg2009** [comment]
Could you please also uploan a screenshot with the dark theme

File: ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift [hunk 1]
```diff
@@ -130,20 +142,30 @@ final class OverviewContactViewController: UIViewController, MFMessageComposeVie
     private func setUpScrollView() {
         view.addSubview(scrollView)
         NSLayoutConstraint.activate([
-            scrollView.topAnchor.constraint(equalTo: view.topAnchor),
-            scrollView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-            scrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
-            scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            scrollView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 8),
```
Comment (pavoleg2009) [inline ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift:145]: nit: it's better to use some constant (ideally - app-wide) for values like this

File: ft_hangouts/ft_hangouts/UserStories/OverviewContact/ProfileHeaderView.swift [hunk 2]
```diff
@@ -38,7 +37,7 @@ final class ProfileHeaderView: UIView {
     // MARK: - Init
     override init(frame: CGRect) {
         super.init(frame: frame)
-        backgroundColor = .secondarySystemBackground
+//        backgroundColor = .tertiarySystemBackground
```
Comment (pavoleg2009) [inline ft_hangouts/ft_hangouts/UserStories/OverviewContact/ProfileHeaderView.swift:40]: please remove any commented code from the PR

**Review by pavoleg2009 — CHANGES_REQUESTED** [review]
Please add a screenshot for the dark theme and remove the commented code

**Comment by hmeriann** [comment]
The commented code removed, dark mode attached
Screenshot 2024-04-15 at 13 46 32

**Review by pavoleg2009 — APPROVED** [review]

**Pull request merged**